doc/userdoc/nest_server.rst

```diff
@@ -0,0 +1,649 @@
+NEST Server
+===========
+
+.. contents:: On this page, you'll find
+   :local:
+   :depth: 1
+
+What is NEST Server?
+--------------------
+
+NEST Server enables users to interact with the NEST simulation engine
+via a RESTful API. Using this approach, you can perform the same basic
+operations as with :doc:`PyNEST <ref_material/pynest_apis>`, but
+instead of doing so by directly importing the ``nest`` module, all
+commands as well as their arguments and result data are channeled
+through HTTP requests and responses transmitted over a TCP/IP
+connection.
```

Comment:
```suggestion
via a RESTful API. Using this approach, you can perform the same basic
operations as with :doc:`PyNEST <ref_material/pynest_apis>`, but
instead of doing so by directly importing the ``nest`` module, all
commands, including their arguments and result data, are channeled
through HTTP requests and responses transmitted over a TCP/IP
connection.
```

Reply:
Do you need the word transmitted (line 16)? It reads awkwardly to me, and I think it makes sense without the word.